kalbarri-workshop.md

```diff
@@ -144,8 +144,8 @@ var z = x + y;
 </details>
 <br>
 
-### Boolean
-
+### Boolean 
+<!-- Relevant to later exercises? -->
```

Comment:
Not critical to remote sensing exercise later in lab - could be slimmed out saved for a later workshop